kolibri/plugins/coach/assets/src/views/plan/GroupEnrollPage/index.vue

```diff
@@ -0,0 +1,274 @@
+            v-else
+            name="chevron_left"
+            category="navigation"
+          />
+        </UiIconButton>
+        <UiIconButton
+          type="primary"
+          :ariaLabel="$tr('nextResults')"
+          :disabled="pageNum === numPages"
+          size="small"
+          @click="goToPage(pageNum + 1)"
+        >
+          <mat-svg
+            v-if="isRtl"
+            name="chevron_left"
+            category="navigation"
+          />
+          <mat-svg
+            v-else
+            name="chevron_right"
+            category="navigation"
+          />
+        </UiIconButton>
+      </nav>
+
+      <div class="footer">
+        <KButton
+          :text="$tr('confirmSelectionButtonLabel')"
+          :primary="true"
+          type="submit"
+          :disabled="selectedUsers.length === 0"
+        />
+      </div>
+    </form>
+
+
+  </CoreBase>
+
+</template>
+
+
+<script>
+
+  import { mapActions, mapState } from 'vuex';
+  import differenceWith from 'lodash/differenceWith';
+  import UiAlert from 'keen-ui/src/UiAlert';
+  import responsiveWindow from 'kolibri.coreVue.mixins.responsiveWindow';
+  import KButton from 'kolibri.coreVue.components.KButton';
+  import UiIconButton from 'kolibri.coreVue.components.UiIconButton';
+  import KFilterTextbox from 'kolibri.coreVue.components.KFilterTextbox';
+  import commonCoach from '../../common';
+  import {
+    userMatchesFilter,
+    filterAndSortUsers,
+  } from '../../../../../../facility_management/assets/src/userSearchUtils';
+  import UserTable from '../../../../../../facility_management/assets/src/views/UserTable.vue';
+
+  export default {
+    name: 'GroupEnrollPage',
+    components: {
+      KButton,
+      UiIconButton,
+      KFilterTextbox,
+      UiAlert,
+      UserTable,
+    },
+    metaInfo() {
+      return {
+        title: this.$tr('enrollLearnersIntoGroup', { groupName: this.currentGroup.name }),
+      };
+    },
+    mixins: [responsiveWindow, commonCoach],
+    $trs: {
+      enrollLearnersIntoGroup: `Enroll learners into '{groupName}'`,
+      groupsHeader: 'Groups',
+      confirmSelectionButtonLabel: 'Confirm',
+      searchForUser: 'Search for a user',
+      userIconColumnHeader: 'User Icon',
+      name: 'Full name',
+      username: 'Username',
+      userTableLabel: 'User List',
+      role: 'Role',
+      // TODO clarify empty state messages after string freeze
+      noUsersExist: 'No users exist',
+      noUsersSelected: 'No users are selected',
+      noUsersMatch: 'No users match',
+      previousResults: 'Previous results',
+      nextResults: 'Next results',
+      selectAllOnPage: 'Select all on page',
+      allUsersAlready: 'All users are already enrolled in this group',
+      search: 'Search',
+      selectUser: 'Select user',
+      addUsersError: 'There was a problem adding users to this group',
+      pagination:
+        '{ visibleStartRange, number } - { visibleEndRange, number } of { numFilteredUsers, number }',
```

Comment:
looks like we have some string freeze issues here? As far as I can tell these strings are being attached to a new namespace?

In order to get around string freeze, we need to grab the strings from a location that is already translated.

To do this, you can either use the `crossComponentTranslator`, or create a new javascript translator and manually specify an old component namespace.